### DISCUSSION TITLE
NAS for NLP

### DISCUSSION THREAD

**Discussion by ShashwatVv** [microsoft/nni#5066] [Q&A]
Hello folks,
Can one of you please point out some actual code-based implementation of NAS in the domain of NLP?
Thank you.

**Comment by ultmaster** [comment]
The current implementation of NNI can be smoothly extended to different tasks, including NLP. I didn't see why that's a problem.

Lacking of related examples could be a problem. You might need to implement search spaces like NAS-BERT on your own. :(
